### PR TITLE
made the steps "optional"

### PR DESCRIPTION
**Pull request by johnpapa**
as per feedback from meeting with the review team and Scott, these are not steps, but rather optional things to install.

#sign-off

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/node-azure-tools.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-2/docs-ref-conceptual/node-azure-tools.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C1%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201708011621071220-176%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-conceptual/node-azure-tools.md [hunk 1]
```diff
@@ -13,16 +13,16 @@ ms.author: tarcher
 ---
 
 # Azure tools for Node.js developers
-Whatever your operating system, we have a great set of recommended tools to help you develop using Node.js on Azure.
+We recommend these great tools to develop using Node.js on Azure, whatever your operating system.
```
Comment (burkeholland) [inline docs-ref-conceptual/node-azure-tools.md]: Proposed...

We recommend these great tools to develop using Node.js on Azure, no matter what your operating system is.

**Comment by burkeholland** [comment]
#sign-off per @TomArcher

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/node-azure-tools.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-2/docs-ref-conceptual/node-azure-tools.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C1%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201708011648164732-176%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**